docs/querying-data.rst

```diff
@@ -0,0 +1,157 @@
+Querying data in Kingfisher Process
+====================================
+
+The Kingfisher Process :doc:`data-model` is based on collections. Each collection contains either data downloaded from an OCDS publisher using a `Kingfisher Scrape <https://kingfisher-scrape.readthedocs.io/en/latest/>`__ spider, or loaded locally using the :doc:`cli/local-load` cli tool.
+
+OCDS data can be published using either `releases or records <https://standard.open-contracting.org/latest/en/getting_started/releases_and_records/>`__. Releases are point in time updates about a contracting process and a record provides an index of all the releases for a contracting process.
+
+Kingfisher Process automatically transforms OCDS releases to create compiled release collections,  containing a single compiled release per contracting process populated with the latest value of each field.
```

Comment:
Suggest merging this into home page.

Reply:
Done in https://github.com/open-contracting/kingfisher-process/pull/220/commits/f147f690aec9099b7e405943b55a780027dc9ee4